src/Multipart/AbstractUploadBuilder.php

```diff
@@ -9,7 +9,7 @@
 abstract class AbstractUploadBuilder
```

Comment:
I suggest either marking this class as internal or refactoring it to use private properties with protected accessor methods.